### ISSUE TITLE
Hospitality: Visitors from X are leaving pinging spam

### ISSUE THREAD

**Issue by Flavio11113** [OrionFive/Hospitality#510]
**Describe the bug**
Title. Pretty much had guests(robots) that wouldn't want to leave after some time, passed then into the world thinking that is a safe way to do stuff and about 30 min later I get spam pinged by the same faction leaving.

I have empire installed, but it didn't have any problems with it at all yet. Uninstalling it doesn't solve the problem! Maybe its only because I force pass them to world. 

Could you give me a solution how I can manually fix this group, have read #500 but I don't quite understand how I can attend your "fix". Thanks 👍

**To Reproduce**
1. Pass to world with dev tool
2. Wait a long time(???)
3. profit

**Console log**
https://gist.github.com/aed98f2f0c3c24f2790129d77e062e8a

**Error stacktrace**
Exception while ticking lord with job Hospitality.LordJob_VisitColony: 
System.NullReferenceException: Object reference not set to an instance of an object
  at Hospitality.TransitionAction_EnsureHaveNearbyExitDestination.DoAction (Verse.AI.Group.Transition trans) [0x0007c] in <9feb82f59fe74b2c90edd9330bfc28c6>:0 
  at (wrapper dynamic-method) Verse.AI.Group.Transition.Verse.AI.Group.Transition.Execute_Patch1(Verse.AI.Group.Transition,Verse.AI.Group.Lord)
  at Verse.AI.Group.Transition.CheckSignal (Verse.AI.Group.Lord lord, Verse.AI.Group.TriggerSignal signal) [0x000de] in <d72310b4d8f64d25aee502792b58549f>:0 
  at Verse.AI.Group.Lord.CheckTransitionOnSignal (Verse.AI.Group.TriggerSignal signal) [0x00050] in <d72310b4d8f64d25aee502792b58549f>:0 
  at Verse.AI.Group.Lord.LordTick () [0x0002a] in <d72310b4d8f64d25aee502792b58549f>:0 
  at Verse.AI.Group.LordManager.LordManagerTick () [0x00011] in <d72310b4d8f64d25aee502792b58549f>:0 
Verse.Log:Error(String, Boolean)
Verse.AI.Group.LordManager:LordManagerTick()
Verse.Map:Verse.Map.MapPostTick_Patch1(Map)
Verse.TickManager:DoSingleTick()
Verse.TickManager:TickManagerUpdate()
Verse.Game:Verse.Game.UpdatePlay_Patch1(Game)
Verse.Root_Play:Verse.Root_Play.Update_Patch1(Root_Play)
![20210113110812_1](https://user-images.githubusercontent.com/53064944/104437930-b1680280-558f-11eb-8f39-26fbfe4f0131.jpg)

**Screenshots**
If you think it helps, add screenshots to help explain the problem.

**Comment by Flavio11113** [comment]
Otherwise I have to use a backup from a hour earlier

**Comment by Flavio11113** [comment]
Deactivated, saved on a alternative save, then activated Hospitality, the ping spam is now over, but the console is being spam with a new problem.

System.NullReferenceException: Object reference not set to an instance of an object
  at Verse.AI.Group.LordManager.LordManagerTick () [0x00034] in <d72310b4d8f64d25aee502792b58549f>:0 
  at (wrapper dynamic-method) Verse.Map.Verse.Map.MapPostTick_Patch1(Verse.Map)
Verse.Log:Error(String, Boolean)
Verse.Map:Verse.Map.MapPostTick_Patch1(Map)
Verse.TickManager:DoSingleTick()
Verse.TickManager:TickManagerUpdate()
Verse.Game:Verse.Game.UpdatePlay_Patch1(Game)
Verse.Root_Play:Verse.Root_Play.Update_Patch1(Root_Play)

Maybe some leftover stuff the mod left? Should I even continue play on the alternative save without having to worry that my game will break somehow?

**Comment by Flavio11113** [comment]
Even with hospitality disabled the error above still persists

**Comment by OrionFive** [comment]
Removing mods midgame is not supported by RimWorld. So doing that will cause new errors.
Passing guests to world will create the error you are describing.
The only way you can fix it is find the group in your save file and remove it manually.

**Comment by Flavio11113** [comment]
Didn't know that removing mods midgame can lead into new errors. 

Because I got no introduction from you how to remove groups manually, I looked into the corrupted save file and found out myself by wild guessing what lines I had to remove exactly.

The group only consisted of androids, means I searched for an android only pawn group and remove the lines like in the screenshot. 

I explain it here for future people having the same problems and have no clue what to touch in the file. That worked out for me 👍.

Search for lord manager with STRG + F

![43242342343242](https://user-images.githubusercontent.com/53064944/104607302-fa4bb400-5680-11eb-92f5-c45440bc6d3e.PNG)
